addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -295,7 +303,31 @@ func _ready():
 			_movable.last_scale = scale
 			_movable.update_terrain()
 
+func connect_trigger_events():
+	if not is_trigger:
+		return
+	if not self.is_connected("area_entered", self, "element_entered"):
+		connect("area_entered", self, "element_entered")
+	if not self.is_connected("area_exited", self, "element_exited"):
+		connect("area_exited", self, "element_exited")
+	if not self.is_connected("body_entered", self, "element_entered"):
+		connect("body_entered", self, "element_entered")
+	if not self.is_connected("body_exited", self, "element_exited"):
+		connect("body_exited", self, "element_exited")
+
+func disconnect_trigger_events():
+	if not is_trigger:
+		return
```

Comment:
Similar question here. Also: Where is this method called? I only see this definition but no invocation of it anywhere.

Reply:
Hmm, that's right. That reminds me that I was working on a bug happening when loading a game while player character is in the middle of a trigger area: upon loading, the `trigger_out` event would be fired. These functions here were created on this purpose. I'll check again if that bug is still present.

Reply:
Fixed in last commit! 🥳